props/saber_fett263_buttons.h

```diff
@@ -355,6 +1485,308 @@ SaberFett263Buttons() : PropBase() {}
+    MENU_MAX_CLASH,
+    COLOR_BASE,
+    COLOR_ALT,
+    COLOR_BLAST,
+    COLOR_CLASH,
+    COLOR_LOCKUP,
+    COLOR_DRAG,
+    COLOR_LB,
+    COLOR_STAB,
+    COLOR_PREON,
+    COLOR_IGNITE,
+    COLOR_RETRACT,
+    COLOR_PSTOFF,
+    COLOR_SWING,
+    COLOR_EMITTER,
+    COLOR_OFF,
+#endif  
+  };
+
+  enum EditColorMode {
+    NONE,
+    CC_COLOR_LIST,
+#ifdef FETT263_EDIT_MODE_MENU
+    COLOR_LIST,
+    EDIT_BLACK,
+    EDIT_COLOR,
+    EDIT_WHITE,
+    ZOOM_COLOR,
+#endif
+  };
+
+#define CC_NEW_COLOR(N) NewColor(N,1);
+
+// Update Style
+  virtual void UpdateStyle(int preset_num) {
+    TRACE(PROP, "start");
+    SaveColorChangeIfNeeded();
+    // First free all styles, then allocate new ones to avoid memory
+    // fragmentation.
+#define UNSET_BLADE_STYLE(N) \
+    delete current_config->blade##N->UnSetStyle();
+    ONCEPERBLADE(UNSET_BLADE_STYLE)
+    current_preset_.SetPreset(preset_num);
+#ifdef DYNAMIC_BLADE_LENGTH
+    savestate_.ReadINIFromSaveDir("curstate");
+#define WRAP_BLADE_SHORTERNER(N) \
+    if (savestate_.blade##N##len != -1 && savestate_.blade##N##len != current_config->blade##N->num_leds()) { \
+      tmp = new BladeShortenerWrapper(savestate_.blade##N##len, tmp);	\
+    }
+#else
+#define WRAP_BLADE_SHORTERNER(N)
+#endif
+
+    
+#define SET_BLADE_STYLE(N) do {						\
+    BladeStyle* tmp = style_parser.Parse(current_preset_.current_style##N.get()); \
+    WRAP_BLADE_SHORTERNER(N)                                            \
+    current_config->blade##N->SetStyle(tmp);				\
+  } while (0);
+
+    ONCEPERBLADE(SET_BLADE_STYLE)
+
+#ifdef SAVE_COLOR_CHANGE
+    SaberBase::SetVariation(current_preset_.variation);
+#else
+    SaberBase::SetVariation(0);
+#endif
+    TRACE(PROP, "end");
+  }
+
+// Update Font/ Save Style
+  virtual void UpdateFont(int preset_num) {
+    TRACE(PROP, "start");
+    bool on = SaberBase::IsOn();
+    if (on) Off();
+    SaveColorChangeIfNeeded();
+    // First free all styles, then allocate new ones to avoid memory
+    // fragmentation.
+#define UNSET_BLADE_STYLE(N) \
+    delete current_config->blade##N->UnSetStyle();
+    ONCEPERBLADE(UNSET_BLADE_STYLE)
+    current_preset_.SetPreset(preset_num);
+#define SET_BLADE_STYLE(N) \
+    current_config->blade##N->SetStyle(style_parser.Parse(current_preset_.current_style##N.get()));
+    ONCEPERBLADE(SET_BLADE_STYLE)
+    chdir(current_preset_.font.get());
+#ifdef SAVE_COLOR_CHANGE
+    SaberBase::SetVariation(current_preset_.variation);
+#else
+    SaberBase::SetVariation(0);
+#endif
+    if (on) {
+#ifdef FETT263_EDIT_MODE_MENU
+      if (menu_type_ == MENU_PREON_OPTION || menu_type_ == MENU_PREON_SIZE) {
+        On();
+      } else {
+        FastOn();
+      }
+#else
+      FastOn();
+#endif      
+    }
+    TRACE(PROP, "end");
+  }
+
+  bool Parse(const char *cmd, const char* arg) override {
+    if (PropBase::Parse(cmd, arg)) return true;
+    if (!strcmp(cmd, "list_current_tracks")) {
+      // Tracks must be in: font/tracks/*.wav 
```

Comment:
Most people have their tracks in /tracks/

Reply:
this is explained in the documentation, if users want a universal tracks/ folder then can add to "common" in the Edit Mode set up. this was done intentionally to allow fonts to have dedicated tracks for the Track Player rather than end up with the player playing unrelated tracks for characters or themes.